### PR TITLE
fix(readme): fix README image link

### LINKED ISSUE #4483: Broken image in README
## Possible Solution
I don't remember what this image used to be so I don't know where it's supposed to come from

### Versions
`master`

### PR DESCRIPTION
**Pull request by machikoyasuda**
Resolves #4483
Impact: **minor**  
Type: **docs**

## Issue
- README image is a dead link

## Solution
- Use correct link from here: https://github.com/reactioncommerce/reaction-docs/blob/v1.7.0/assets/Reaction-Commerce-Illustration-BG-800px.png

## Breaking changes
None

## Testing
1. View https://github.com/reactioncommerce/reaction/blob/c12c6df176cce59c69e944910f7f1506daeeb706/README.md

### CODE REVIEW

**Comment by zenweasel** [comment]
I wonder if we shouldn't fix the fact that we still say "Out of the box features include: Analytics." When we don't actually include any Analytics out of the box.

**Comment by zenweasel** [comment]
We don't really have "Drag and Drop merchandising" any more

**Comment by zenweasel** [comment]
And the "Reaction Action RSVP" link takes you to Reaction Action YT videos

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 [Reaction](http://reactioncommerce.com) is an event-driven, real-time reactive commerce platform built with JavaScript (ES6). It plays nicely with npm, Docker, and React.
 
-![Reaction v.1.x](https://raw.githubusercontent.com/reactioncommerce/reaction-docs/master/assets/Reaction-Commerce-Illustration-BG-800px.png)
+![Reaction v.1.x](https://raw.githubusercontent.com/reactioncommerce/reaction-docs/v1.7.0/assets/Reaction-Commerce-Illustration-BG-800px.png)
```
Comment (spencern) [inline README.md]: Should the alt text here say `Reaction v1.x` instead of `Reaction v.1.x`?

**Pull request merged**